<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>633</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Helvetica</family>
   </font>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>25</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Server </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Choosing Index Structure</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Grid-Tree</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kd-Tree</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainexit_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>97</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mainexit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>97</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Interacting With System</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>501</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>461</width>
       <height>211</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>441</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;style&gt;
    .table{
        table-layout: fixed;
    }
   .table td:hover{
       overflow: visible;
       white-space: normal;
   }
    .table td {
        word-wrap: break-word;
        overflow: hidden;
        text-overflow: ellipsis;
        white-space: nowrap;
    }
&lt;/style&gt;
</string>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideRight</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Patient ID</string>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Check-Up No.</string>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Diagnose</string>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Time</string>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End Time</string>
       </property>
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>P0066</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <underline>false</underline>
         <stylestrategy>PreferDefault</stylestrategy>
        </font>
       </property>
       <property name="background">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="0" column="1">
       <property name="text">
        <string>No.2</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="0" column="2">
       <property name="text">
        <string>Direct infection of joint in infectious and parasitic diseases classified elsewhere</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="0" column="3">
       <property name="text">
        <string>1994-07-22 07:12:50.407</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="0" column="4">
       <property name="text">
        <string>1994-07-28 12:29:05.827</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>P0066</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <underline>false</underline>
        </font>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="1" column="1">
       <property name="text">
        <string>No.3</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <underline>false</underline>
        </font>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="1" column="2">
       <property name="text">
        <string>Disseminated intravascular coagulation [defibrination syndrome]</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="1" column="3">
       <property name="text">
        <string>2003-03-02 09:59:11.540</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="1" column="4">
       <property name="text">
        <string>2003-03-22 01:31:32.883</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>P0066</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <underline>false</underline>
        </font>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="2" column="1">
       <property name="text">
        <string>No.4</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="2" column="2">
       <property name="text">
        <string>Chronic myeloid leukemia, BCR/ABL-positive</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="2" column="3">
       <property name="text">
        <string>2007-07-03 20:00:07.223</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="2" column="4">
       <property name="text">
        <string>2007-07-08 14:23:38.367</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>P0066</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <underline>false</underline>
        </font>
       </property>
       <property name="foreground">
        <brush brushstyle="NoBrush">
         <color alpha="255">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </property>
      </item>
      <item row="3" column="1">
       <property name="text">
        <string>No.5</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <underline>false</underline>
         <kerning>true</kerning>
        </font>
       </property>
      </item>
      <item row="3" column="2">
       <property name="text">
        <string>Rheumatoid polyneuropathy with rheumatoid arthritis of shoulder</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="3" column="3">
       <property name="text">
        <string>2012-04-13 13:23:12.527</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="3" column="4">
       <property name="text">
        <string>2012-04-24 21:10:59.800</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>P0000</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="4" column="1">
       <property name="text">
        <string>No.1</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="4" column="2">
       <property name="text">
        <string>Malignant neoplasm</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="4" column="3">
       <property name="text">
        <string>1992-07-01 08:36:21</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
      <item row="4" column="4">
       <property name="text">
        <string>1992-07-24 08:36:21</string>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
        </font>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QPushButton" name="mainexit_3">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>15</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Attack</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
       <pointsize>16</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Database Overview</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>501</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Query Log</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>501</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <zorder>frame_3</zorder>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>label_14</zorder>
   <zorder>label</zorder>
   <zorder>radioButton_1</zorder>
   <zorder>radioButton_2</zorder>
   <zorder>mainexit_2</zorder>
   <zorder>mainexit</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>textBrowser</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
